libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/DefaultAudioTrackBufferSizeProvider.java

```diff
@@ -143,6 +153,17 @@ public Builder setAc3BufferMultiplicationFactor(int ac3BufferMultiplicationFacto
       return this;
     }
 
+    /**
+     * Sets the multiplication factor to apply to the passthrough buffer for DTS-HD (DTS Express)
+     * to avoid underruns on some devices (e.g., Xiaomi A2 TV). Default is
+     * {@value #DTSHD_BUFFER_MULTIPLICATION_FACTOR}.
```

Comment:
nit: `{@link #DTSHD_BUFFER_MULTIPLICATION_FACTOR}` instead of `@value`.

Reply:
Done